packages/smart-contracts/scripts-create2/constructor-args.ts

```diff
@@ -20,9 +20,11 @@ export const getConstructorArgs = (contract: string, network?: string): string[]
       return [
         '0x0000000000000000000000000000000000000000',
         '0x0000000000000000000000000000000000000000',
+        // FIXME: This is not right the NativeTokenHash is not the same accross all networks
```

Comment:
this scares me a bit... don't you think there should be something in the code preventing wrong execution? A FIXME will most likely be forgotten

Reply:
The scripts deploys the contract specified in `script-create2/utils`. `EthConversionProxy` is commented there with a FIXME, so I think it's safe.

Reply:
Maybe raise an error here => nobody can do the error if there is a fast deployment.